docs/semantics/func.md

```diff
@@ -0,0 +1,41 @@
+# Func Semantic
+
+## Overview
+
+The `func` type will reference a function in FIRE. The `func` type can be passed around as a parameter, or called on by other code in a given program. 
+
+## Syntax
+
+#### Declaration
+the `func` type is declared in the following format\:
+`func <name> = (<parameters>) => { <function body> };`
+
+where:  
+ `<name>` is the variable name of the func.  
+ `<paramters>` are the expected parameters for the function
+ `<function body>` is the body of the function.
+ 
+ once a function has been assigined to a `func` type it becomes a "named function" that is callable using that name e.g `funcName();`
```

Comment:
Capital `O` in once and theres a space before the start of the sentence after the new line